src/world/block_textures.rs

```diff
@@ -0,0 +1,19 @@
+pub enum BlockTypes {
+    Air = 0,
+    Grass = 1,
+    Dirt = 2,
+    Stone = 3,
+    Log = 4,
+    Leaves = 5,
+    Planks = 6,
+}
+
+pub const TEXTURE_MAP: [&[[&str; 5]; 5]; 7] = [&AIR, &GRASS, &DIRT, &STONE, &LOG, &LEAVES, &PLANKS];
+
+pub const AIR: [[&str; 5]; 5] = [["#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef"],["#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef"],["#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef"],["#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef"],["#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef","#a3e0ef"]];
```

Comment:
I love this: very clever with no bloat.

Hypothetically, if you were to add more blocks, increase pixels per block blocks, etc. 1. At what point would you start looking at alternatives? and 2. What what would you change this too?